src/main/java/jenkins/scm/impl/subversion/SubversionSCMSource.java

```diff
@@ -159,6 +165,18 @@ public String getCredentialsId() {
     public void setCredentialsId(String credentialsId) {
         this.credentialsId = credentialsId;
     }
+    @DataBoundSetter
+    public void setAdditionalCredentials(List<AdditionalCredentials> additionalCredentials) {
+        this.additionalCredentials = additionalCredentials;
+    }
+    //without getter jelly will crash
+    @Restricted(NoExternalUse.class)
+    public List<AdditionalCredentials> getAdditionalCredentials() {
+        if(additionalCredentials==null) {
+            return Collections.emptyList();
+        }
+        return(additionalCredentials);
```

Comment:
would be better to just `return Collections.unmodifyableList(additionalCredentials)`